docs/performance/tune-performance/threading.md

```diff
@@ -14,75 +14,105 @@ nav_order: 3
 {:toc}
 
 
-For the default CPU execution provider, you can use the following knobs in the Python API to control the thread number:
+For the default CPU execution provider, setting defaults are provided to get fast inference performance. You can customize the performance using the following knobs in the API to control the thread count and other settings:
 
+Python (Defaults):
 ```python
 import onnxruntime as rt
 
 sess_options = rt.SessionOptions()
 
-sess_options.intra_op_num_threads = 2
+sess_options.intra_op_num_threads = 0
 sess_options.execution_mode = rt.ExecutionMode.ORT_SEQUENTIAL
 sess_options.graph_optimization_level = rt.GraphOptimizationLevel.ORT_ENABLE_ALL
+sess_options.AddConfigEntry(kOrtSessionOptionsConfigAllowIntraOpSpinning, "1")
 ```
 
 
-* Thread Count
+* INTRA Thread Count
+
+  * Controls the _total_ number of INTRA threads to use to run the model. 
+  * INTRA = parallelize computation _inside_ each operator
+  * Default: (not specified or 0). `sess_options.intra_op_num_threads = 0`
+    * INTRA Threads Total = Number of physical CPU Cores. Leaving at default also enables some affinitization (explained below)
+    * E.g. 6-core machine (with 12 HT logical processors) = 6 total INTRA threads
 
-  * `sess_options.intra_op_num_threads = 2` controls the number of threads to use to run the model.
 
 * Sequential vs Parallel Execution
 
-  * `sess_options.execution_mode = rt.ExecutionMode.ORT_SEQUENTIAL` controls whether the operators in the graph run sequentially or in parallel. Usually when a model has many branches, setting this option to `ORT_PARALLEL` will provide better performance.
+  * Controls whether _multiple_ operators in the graph (_across_ nodes) run sequentially or in parallel. 
+  * Default: `sess_options.execution_mode = rt.ExecutionMode.ORT_SEQUENTIAL`
+  * Usually when a model has many branches, setting this option to `ORT_PARALLEL` will provide better performance. This could also hurt performance on some models without many branches.
```

Comment:
ORT_PARALLEL is not supported anymore. You can safely omit this part and all references to INTER op threadpool.

Reply:
Also I would prefer ORT team officially deprecates INTER in another PR. I think the scope of this PR is to make clearer the existing supported settings